### ISSUE TITLE
Bring back auto-login feature

### ISSUE THREAD

**Issue by yanus** [cuzy-app/humhub-modules-auth-keycloak#13]
After upgrading recently, I noticed that the automatic login wasn't working anymore and after some investigation realised that the feature had been removed. This is quite important for us, as it enables a smooth user experience. When users are presented with a separate login form now, many of them will be confused and not know whether to click the button or enter their credentials again.
What's the reasoning behind removing this essential feature and is there a chance to get it back?

Cheers,
Yannick

**Comment by marc-farre** [comment]
Yes, this is since [version 1.3.0](https://github.com/cuzy-app/humhub-modules-auth-keycloak/releases/tag/1.3.0).
I had to remove it because it wasn't working anymore on recent browser because of too many automatic redirections (at least on Chrome).
If you want to get it back, we need to find a different solution, which might involve having to modify the HumHub kernel.

So I cannot just put back the old code, it needs working on it. Personally, I don't have the need of it anymore, but you can suggest a code by submitting a pull request or pay a developer to do it.

**Comment by yanus** [comment]
Alright, thanks for the quick response and explanation, Marc!
